Microsoft.Toolkit.Uwp.UI.Controls/TabView/TabView.Properties.cs

```diff
@@ -0,0 +1,214 @@
+        public static readonly DependencyProperty ItemHeaderTemplateProperty =
+            DependencyProperty.Register(nameof(ItemHeaderTemplate), typeof(DataTemplate), typeof(TabView), new PropertyMetadata(null));
+
+        /// <summary>
+        /// Gets or sets a value indicating whether an 'Add' button appears for the user to add tabs themselves.
+        /// </summary>
+        public bool IsAddTabButtonVisible
+        {
+            get { return (bool)GetValue(IsAddTabButtonVisibleProperty); }
+            set { SetValue(IsAddTabButtonVisibleProperty, value); }
+        }
+
+        /// <summary>
+        /// Identifies the <see cref="IsAddTabButtonVisible"/> dependency property.
+        /// </summary>
+        /// <returns>The identifier for the <see cref="IsAddTabButtonVisible"/> dependency property.</returns>
+        public static readonly DependencyProperty IsAddTabButtonVisibleProperty =
+            DependencyProperty.Register(nameof(IsAddTabButtonVisible), typeof(bool), typeof(TabView), new PropertyMetadata(false));
+
+        /// <summary>
+        /// Gets or sets a value indicating whether by default a Tab can be closed or not if no value to <see cref="TabViewItem.IsClosable"/> is provided.
+        /// </summary>
+        public bool CanCloseTabs
+        {
+            get { return (bool)GetValue(CanCloseTabsProperty); }
+            set { SetValue(CanCloseTabsProperty, value); }
+        }
+
+        /// <summary>
+        /// Identifies the <see cref="CanCloseTabs"/> dependency property.
+        /// </summary>
+        /// <returns>The identifier for the <see cref="CanCloseTabs"/> dependency property.</returns>
+        public static readonly DependencyProperty CanCloseTabsProperty =
+            DependencyProperty.Register(nameof(CanCloseTabs), typeof(bool), typeof(TabView), new PropertyMetadata(false));
+
+        /// <summary>
+        /// Gets or sets the implementer of <see cref="ITabWidthProvider"/> interface to provide widths of tabs for <see cref="TabView"/>.
+        /// </summary>
+        public ITabWidthProvider TabWidthProvider
+        {
+            get { return (ITabWidthProvider)GetValue(TabWidthProviderProperty); }
+            set { SetValue(TabWidthProviderProperty, value); }
+        }
+
+        /// <summary>
+        /// Identifies the <see cref="TabWidthProvider"/> dependency property.
+        /// </summary>
+        /// <returns>The identifier for the <see cref="TabWidthProvider"/> dependency property.</returns>
+        public static readonly DependencyProperty TabWidthProviderProperty =
+            DependencyProperty.Register(nameof(TabWidthProvider), typeof(ITabWidthProvider), typeof(TabView), new PropertyMetadata(new ActualTabWidthProvider()));
+
+        public static bool GetIgnoreColumn(ColumnDefinition obj)
+        {
+            return (bool)obj.GetValue(IgnoreColumnProperty);
+        }
+
+        public static void SetIgnoreColumn(ColumnDefinition obj, bool value)
```

Comment:
missing docs

Reply:
Yeah, sorry, this is more of an internal thing, but I should add a comment.

Reply:
It's internal to the logic of the control, but it can't be internalized as then the xaml engine can't find the attached property correctly (at least that's what I've seen in the past with the UniformGrid which needed a similar construct).